doc/ale-idris.txt

```diff
@@ -0,0 +1,10 @@
+===============================================================================
+ALE Idris Integration                                   *ale-idris-options*
+
+===============================================================================
+idris                                                   *ale-idris-stack-build*
```

Comment:
The help tag is wrong here. It should be `ale-idris-idris`, as in the table of contents.